sbt-test/scala3-compat/macros-forward-3.0/build.sbt

```diff
@@ -0,0 +1,13 @@
+lazy val checkOptions = Seq("-Xcheck-macros", "-Ycheck:all", "-Yno-double-bindings")
+
+lazy val lib = project.in(file("lib"))
+  .settings(
+    scalacOptions ++= Seq("-Yscala-release", "3.0") ++ checkOptions
+  )
+
+lazy val app = project.in(file("app"))
+  .dependsOn(lib)
+  .settings(
+    scalaVersion := "3.0.2",
```

Comment:
@smarter I ported the previous tests to `sbt-test`. But it seems that this project is trying to load the `3.1.3-RC1-bin-SNAPSHOT` standard library. This happens when I have a dependency on `lib` (even when `lib` is empty). Do you know what I'm missing? 

```
[info] [info] compiling 1 Scala source to /private/var/folders/gl/4_k_l76n3pq17f7zxm0v6v_w0000gn/T/sbt_11f9b5da/app/target/scala-3.0.2/classes ...
[info] [error] error while loading $throws$package$,
[info] [error] class file scala/runtime/$throws$package.class is broken, reading aborted with class dotty.tools.tasty.UnpickleException
[info] [error] TASTy signature has wrong version.
[info] [error]  expected: {majorVersion: 28, minorVersion: 0}
[info] [error]  found   : {majorVersion: 28, minorVersion: 2 [unstable release: 1]}
[info] [error] 
[info] [error] This TASTy file was produced by an unstable release.
[info] [error] To read this TASTy file, your tooling must be at the same version.
[info] [error] The TASTy file was produced by Scala 3.1.3-RC1-bin-SNAPSHOT-nonbootstrapped-git-17bf27f.
[info] [error] one error found
[info] [error] one error found
[info] [error] (app / Compile / compileIncremental) Compilation failed
[info] [error] Total time: 7 s, completed Feb 3, 2022 5:05:51 PM
````

Reply:
Yeah for now until sbt learns about -Yscala-release you have to manually make the app project depend on the old standard library, see the bottom of https://docs.scala-lang.org/scala3/reference/language-versions/binary-compatibility.html